super/pom.xml

```
@@ -95,13 +95,14 @@
     <maven.enforcer.maven.version>3.8.5</maven.enforcer.maven.version>
     <jaxb2.maven.plugin.version>2.5.0</jaxb2.maven.plugin.version>
     <jaxb.api.version>2.3.1</jaxb.api.version>
+    <jaxb.core.version>2.3.0.1</jaxb.core.version>
```

Could we use newer versions (and switch from javax.xml.bind to jakarta.xml.bind)?
```suggestion
    <jaxb.api.version>4.0.0</jaxb.api.version>
    <jaxb.core.version>4.0.1</jaxb.core.version>
```